starfish/io.py

```diff
@@ -123,3 +124,65 @@ def un_squeeze(self, stack):
         new_shape = (self.image.num_hybs, self.image.num_chs, self.image.num_zlayers) + self.image.tile_shape
         res = stack.reshape(new_shape)
         return res
+
+    def show_z_stack(
+            self, h: int=1, c: int=0, cmap: str='gray', rescale: bool=False, figsize: Tuple[int, int]=(10, 10)):
+        """Create an interactive visualization of a z-stack
+
+        Produces a slider that flips through the z-series of 2-d x-y images
+
+        Parameters
+        ----------
+        h : int
+            hybridization round to visualize
+        c : int
+            channel to visualize
+        cmap : str (default = 'gray')
+            string id of a matplotlib colormap
+        rescale : bool (default = True)
+            if True, rescale the data to exclude high and low-value outliers (see skimage.exposure.rescale_intensity)
+        figsize : Tuple[int, int] (default = (10, 10))
+            size of the figure in inches
+
+        Notes
+        -----
+        For this widget to function interactively in the notebook, after ipywidgets has been installed, the user must
+        register the widget with jupyter by typing the following command into the terminal:
+        jupyter nbextension enable --py widgetsnbextension
+
+        """
+        from skimage import exposure
```

Comment:
Because we're now type-annotating, we can't really avoid importing the heavy libraries at load time.  There's no advantage to doing these lazy imports any more.